app/routes/user.py

```diff
@@ -519,4 +522,30 @@ def user_signup_fake():
   db.session.add(user)
   db.session.commit()
   user.id
-  return user.get_dict()
\ No newline at end of file
+  return user.get_dict()
+
+@bp.route('/<int:id>/upload', methods=['POST'])
+@jwt_required()
+def upload(id):
+  if 'file' not in request.files:
+    return { 'msg': 'No file included in request' }, 422
+  # If the user does not select a file, the browser submits an
+  # empty file without a filename.
+  file = request.files.get('file')
+  if file.filename == '':
+      return { 'msg': 'Submitted an empty file' }, 422
+  import uuid
+  s3_key = str(get_current_user().id) + '_' + str(uuid.uuid4())
+  contents = file.read()
+  bucket = os.environ.get('S3_BUCKET_NAME')
+  s3_url = boto3.client("s3").upload_fileobj(io.BytesIO(contents), bucket, 'users/'+s3_key,
+                            ExtraArgs={'ACL': 'public-read',
+                                        'ContentType': file.content_type}
+                            )
+  s3_url = f'https://{bucket}.s3.amazonaws.com/users/{s3_key}'
```

Comment:
change this folder name from `/users/` to `/profile_pic/`. it actually already exists for some users where i manually set their image